### ISSUE TITLE
Error During Save: Null

### ISSUE THREAD

**Issue by terrigan** [pwm-project/pwm#481]
New user using the most recent build 2019-04-16T07_21_53Z on an Ubuntu VM following the [instructions](https://github.com/pwm-project/pwm/wiki/Installation-on-Ubuntu-Server-16.04-LTS) on the wiki gets completely through the configuration with all green checks on testing connectivity to get to the Summary screen to receive a 5015 ERROR_INTERNAL (error during save: null). The only deviation from the instructions is using the current Ubuntu 18.04.2 LTS Server image vs the older 16.

**Comment by liu35118665** [comment]
today me too @terrigan

**Comment by liu35118665** [comment]
frontend show:

> An error has occurred. If this error occurs repeatedly please contact your help desk.

> 5015 ERROR_INTERNAL (error during save: null)

console logs:
`2019-04-29T13:49:20Z, TRACE, http.PwmRequest, {LLAh0} POST request for: /pwm/private/config/config-guide completed requestID=2796 in 212ms [0:0:0:0:0:0:0:1]
  preventCache='1556516960556'
  processAction='gotoStep'
  step='FINISH'
  pwmFormID='H4sIAAAAAAAAAAGaAGX_UFdNLkdDTTEQsP6tta7LYl3YK0sG94onAMo4FLguAcZdHzPYLtA8Cvz-l57bbAVCUoGOy2bcbGc6LlJrKoQnKd-Kf29qfakdqNrWntbijYfuhYQ2HswTf5zEXX3jS8T4LdTmACI0wEb6HZ8blQWMIbwwgJkUDWL-CxUzfjToQkEvWlPWOWcFzgEZWghtLu8cycf2wRf6mFdY8LQc9JSaAAAA'
2019-04-29T13:49:43Z, TRACE, http.PwmRequest, {LLAh0} POST request for: /pwm/private/config/config-guide requestID=2797 [0:0:0:0:0:0:0:1]
  preventCache='1556516983066'
  processAction='gotoStep'
  step='FINISH'
  pwmFormID='H4sIAAAAAAAAAAGaAGX_UFdNLkdDTTEQsP6tta7LYl3YK0sG94onAMo4FLguAcZdHzPYLtA8Cvz-l57bbAVCUoGOy2bcbGc6LlJrKoQnKd-Kf29qfakdqNrWntbijYfuhYQ2HswTf5zEXX3jS8T4LdTmACI0wEb6HZ8blQWMIbwwgJkUDWL-CxUzfjToQkEvWlPWOWcFzgEZWghtLu8cycf2wRf6mFdY8LQc9JSaAAAA'
2019-04-29T13:49:43Z, WARN , config.Configuration, 5055 ERROR_INVALID_SECURITY_KEY (Security Key value is not configured,will generate temp value for use by runtime instance)
2019-04-29T13:49:43Z, WARN , config.Configuration, 5055 ERROR_INVALID_SECURITY_KEY (Security Key value is not configured,will generate temp value for use by runtime instance)
2019-04-29T13:49:43Z, TRACE, state.CryptoCookieLoginImpl, {LLAh0} wrote LoginInfoBean={"a":false,"p":"*hidden*","t":"UNAUTHENTICATED","af":[],"rq":"2019-04-29T05:49:43Z","g":"jv1xs3265jYLh45bJX8TOhs6028m6dslL7phtUhcsdjAMuokDM77tTcgzSshq5omcWkN1e71","c":0,"lf":[]} [0:0:0:0:0:0:0:1]
2019-04-29T13:49:43Z, TRACE, http.PwmRequest, {LLAh0} POST request for: /pwm/private/config/config-guide completed requestID=2797 in 215ms [0:0:0:0:0:0:0:1]
  preventCache='1556516983066'
  processAction='gotoStep'
  step='FINISH'
  pwmFormID='H4sIAAAAAAAAAAGaAGX_UFdNLkdDTTEQsP6tta7LYl3YK0sG94onAMo4FLguAcZdHzPYLtA8Cvz-l57bbAVCUoGOy2bcbGc6LlJrKoQnKd-Kf29qfakdqNrWntbijYfuhYQ2HswTf5zEXX3jS8T4LdTmACI0wEb6HZ8blQWMIbwwgJkUDWL-CxUzfjToQkEvWlPWOWcFzgEZWghtLu8cycf2wRf6mFdY8LQc9JSaAAAA'`

**Comment by liu35118665** [comment]
in password.pwm.config.stored.StoredConfigurationImpl line 249
`
propertyElement.detach();
`
propertyElement is null. i modify to 
`
if (propertyElement != null) {
     propertyElement.detach();
}
`

problem resolve

**Comment by terrigan** [comment]
I have no idea how or where to edit that within the .war file. I did some
research on the pwm-general google group and it seems every build past the
February 2019 has issues. I am running that build successfully.

**Comment by jrivard** [comment]
This is fixed in the most recent build.
